### PR TITLE
Added missing QE identity functions to windows quote provide DLL.

### LINKED ISSUE #86: QE identity info functions are missing in the windows quote provider dll.
In version 0.0.2 of the Azure-DCAP-Client for windows, the quote provider dll is missing the following functions:

    sgx_get_qe_identity_info
    sgx_free_qe_identity_info

I fix is to update the dcap_provider.def to include these 2 missing functions.

### PR DESCRIPTION
**Pull request by jazzybluesea**
Fixes #86 

Updated windows dll to include previously missing QE identity info functions:

sgx_get_qe_identity_info
sgx_free_qe_identity_info

### CODE REVIEW

File: src/Windows/dll/dcap_provider.vcxproj [hunk 1]
```diff
@@ -15,7 +15,7 @@
     <ProjectGuid>{AAA37DF8-87A5-40E9-B0DF-8919D5E92533}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
-    <!-- The library name is dicatated by Intel. They look, specifically, for a dll
+    <!-- The library name is dicatated by Intel. They look, specifically, for a dll
```
Comment (msft-jonnadul) [inline src/Windows/dll/dcap_provider.vcxproj]: Were these changes intentional or Visual Studios generated? If not can you please revert them?

File: src/Windows/GeneratePackage/Azure.DCAP.Windows.nuspec [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
     <metadata>
         <id>Azure.DCAP.Windows</id>
-        <version>0.0.2</version>
+        <version>0.0.3</version>
         <!-- Authors contain text that appears directly on the gallery -->
         <authors>Microsoft</authors>
         <owners>Microsoft</owners>
```
Comment (msft-jonnadul) [inline src/Windows/GeneratePackage/Azure.DCAP.Windows.nuspec]: Can you also update releaseNotes?

File: src/Windows/dll/dcap_provider.vcxproj (hunk 1 above)
Comment (jazzybluesea) [reply src/Windows/dll/dcap_provider.vcxproj]: These were generated by Visual Studio.

File: src/Windows/GeneratePackage/Azure.DCAP.Windows.nuspec (hunk 2 above)
Comment (jazzybluesea) [reply src/Windows/GeneratePackage/Azure.DCAP.Windows.nuspec]: Sorry, I am not clear which file contains the release notes.  Which file are you referring to?

File: src/Windows/dll/dcap_provider.vcxproj (hunk 1 above)
Comment (msft-jonnadul) [reply src/Windows/dll/dcap_provider.vcxproj]: Can we revert these changes then?

File: src/Windows/GeneratePackage/Azure.DCAP.Windows.nuspec (hunk 2 above)
Comment (msft-jonnadul) [reply src/Windows/GeneratePackage/Azure.DCAP.Windows.nuspec]: Within the same .nuspec file there is a releasenotes section were you should elaborate on the changes made to this release of the Azure DCAP for Windows. For ex: Exporting function calls for QEID querying etc.

File: src/Windows/dll/dcap_provider.vcxproj (hunk 1 above)
Comment (jazzybluesea) [reply src/Windows/dll/dcap_provider.vcxproj]: Honestly, I don't think we should since the next time someone opens the project using VS this changes will apply again.

File: src/Windows/dll/dcap_provider.vcxproj [hunk 3]
```diff
@@ -111,9 +108,9 @@
   <Target Name="GetPrerequisitesFromIntel" BeforeTargets="PrepareForBuild">
     <Exec Command="powershell ..\get-prereqs.ps1" />
   </Target>
-  <!--Target Name="RunUnitTest" AfterTargets="Build" DependsOnTargets="GetPrerequisitesFromIntel;EnsureNuGetPackageBuildImports">
-    <Exec ContinueOnError="True" Command="xcopy /y ..\UnitTest\$(Platform)\$(Configuration)\dcap_provider_ut.exe $(OutDir)"/>
-    <Exec Command="$(OutDir)\dcap_provider_ut.exe" />
+  <!--Target Name="RunUnitTest" AfterTargets="Build" DependsOnTargets="GetPrerequisitesFromIntel;EnsureNuGetPackageBuildImports">
+    <Exec ContinueOnError="True" Command="xcopy /y ..\UnitTest\$(Platform)\$(Configuration)\dcap_provider_ut.exe $(OutDir)"/>
+    <Exec Command="$(OutDir)\dcap_provider_ut.exe" />
```
Comment (msft-jonnadul) [inline src/Windows/dll/dcap_provider.vcxproj]: Can you please remove commented out code?

File: src/Windows/dll/dcap_provider.vcxproj (hunk 1 above)
Comment (msft-jonnadul) [reply src/Windows/dll/dcap_provider.vcxproj]: Outside of the explicitly commented out code, the rest of it seems to be spacing related which is fine for now. Moving forward we do expect all contributors to have VS beautification settings to be turned off.

File: src/Windows/dll/dcap_provider.vcxproj (hunk 3 above)
Comment (jazzybluesea) [reply src/Windows/dll/dcap_provider.vcxproj]: These were commented out to begin with.  I am not sure why it needs to be uncommented in this PR?

File: src/Windows/dll/dcap_provider.vcxproj (hunk 1 above)
Comment (jazzybluesea) [reply src/Windows/dll/dcap_provider.vcxproj]: If that is the case we should clearly document this requirement.

**Comment by msft-jonnadul** [comment]
LGMT, however I'd like to see another PR that addresses some of the code quality changes request here.
